css/style.css

```diff
@@ -748,139 +765,252 @@ footer nav {
+	}
+
+	.image-la {
+		margin-top: var(--spacing-small);
+	}
+
+	.image-la img {
+		height: 50px;
+	}
+}
+
+@media screen and (max-width: 900px) {
+	/* img url update to mobile */
+	.form-text-container {
+		background-image: url('../assets/mobile/contact-form.png');
+	}
+	.form-container {
+		/* change from row to column */
+		flex-direction: column;
+	}
+
+	.form-text-container {
+		width: 100%;
+		min-height: 430px;
+	}
+
+	.form-text-container p {
+		/* text to fit screen */
+		width: 100%;
+	}
+
+	.name-input-container {
+		flex-direction: column;
+	}
 }
 
 /* Phone */
 @media screen and (max-width: 480px) {
-  /* adjusting variables */
-  :root {
-    /* increasing wrapper width - to take up more screen */
-    --wrapper-width: 90%;
-
-    /* adjusting font-sizes */
-    --font300: 1.25rem; /* 20px */
-    --font500: 1.68rem; /* 26.8px */
-    --font600: 1.93rem; /* 30.8px */
-    --font900: 2.75rem; /* 44px */
-  }
-
-  /* Header styles */
-  /* adding space between socials icons and logo */
-  .header-nav {
-    gap: 20px;
-  }
-  /* social icons */
-  .social-icons svg {
-    width: 32px;
-    height: 32px;
-  }
-
-  /* Who We Are styles */
-  .who-we-are .overlay-who {
-    padding-top: 95px;
-  }
-
-  .ecosystem-banner {
-    min-height: 150px;
-  }
-  /* reduced the indent */
-  blockquote {
-    padding-left: var(--spacing-medium);
-    margin-bottom: var(--spacing-large);
-  }
-  /* reduced the size of the - before the blockquote */
-  .solid-line-indent::before {
-    width: 16px;
-    top: 6%;
-    left: 2%;
-  }
-
-  /* adjusting hr in who we are */
-
-  .who-we-are hr {
-    margin-top: var(--spacing-medium);
-  }
-  /* footer styles */
-
-  /* Land Acknowledge button placement */
-  .btn-la {
-    margin: -50px -120px 0 0;
-  }
+	adjusting variables :root {
+		/* increasing wrapper width - to take up more screen */
+		--wrapper-width: 90%;
+
+		/* adjusting font-sizes */
+		--font200: 1rem; /* 16px */
+		--font300: 1.25rem; /* 20px */
+		--font500: 1.68rem; /* 27px */
+		--font600: 1.93rem; /* 31px */
+		--font900: 2.75rem; /* 44px */
+
+		/* adjusting line-heights */
+		--line-height-lg: 2rem;
+	}
+
+	/* Header styles */
+	/* adding space between socials icons and logo */
+	.header-nav {
+		gap: var(--gap-md);
+	}
+	/* social icons */
+	.social-icons svg {
+		width: 32px;
+		height: 32px;
+	}
+
+	/* Who We Are styles */
+	.who-we-are .overlay-who {
+		padding-top: 95px;
+	}
+
+	.ecosystem-banner {
+		min-height: 150px;
+	}
+	/* reduced the indent */
+	blockquote {
+		padding-left: var(--spacing-medium);
+		margin-bottom: var(--spacing-large);
+	}
+	/* reduced the size of the - before the blockquote */
+	.solid-line-indent::before {
+		width: 16px;
+		top: 6%;
+		left: 2%;
+	}
+
+	/* adjusting hr in who we are */
+
+	.who-we-are hr {
+		margin-top: var(--spacing-medium);
+	}
+
+	.modal-scroll {
+		max-height: calc(100vh - 260px);
+	}
+
+	/* footer styles */
+
+	/* Land Acknowledge button placement */
+	.btn-la {
```

Comment:
button could be positioned a little bit to be centered

Reply:
We just updated it, I think it's looking better

<img width="414" alt="Screen Shot 2023-12-11 at 12 27 25 PM" src="https://github.com/indigenousfriends/itlct-website/assets/704697/cf5715a2-7f64-46dc-b04e-a12471d90c2f">